code/modules/hydroponics/plants_weed.dm

```diff
@@ -203,11 +203,11 @@ ABSTRACT_TYPE(/datum/plant/weed)
 		if (POT.growth >= (P.harvtime + DNA.harvtime + 50) && prob(10) && !src.exploding)
 			src.exploding = 1
 			POT.visible_message("<span class='alert'><b>[POT]</b> begins to bubble and expand!</span>")
-			playsound(POT.loc, "sound/effects/bubbles.ogg", 50, 1)
+			playsound(POT, "sound/effects/bubbles.ogg", 50, 1)
```

Comment:
get turf?

Reply:
Commenting for completeness:
playsound calls get_turf() on first parameter